hardhat/infra/ava/conf/lilypad.json

```diff
@@ -0,0 +1,30 @@
+    "CustomVMBuildScript": "",
+    "TeleporterReady": true,
+    "TeleporterKey": "cli-teleporter-deployer",
+    "TeleporterVersion": "v1.0.0",
+    "RunRelayer": true,
+    "SubnetEVMMainnetChainID": 0
+  },
+  "Genesis": "ewogICJhaXJkcm9wQW1vdW50IjogbnVsbCwKICAiYWlyZHJvcEhhc2giOiAiMHgwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwIiwKICAiYWxsb2MiOiB7CiAgICAiMTVkMzRhYWY1NDI2N2RiN2Q3YzM2NzgzOWFhZjcxYTAwYTJjNmE2NSI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiM2M0NGNkZGRiNmE5MDBmYTJiNTg1ZGQyOTllMDNkMTJmYTQyOTNiYyI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiNzA5OTc5NzBjNTE4MTJkYzNhMDEwYzdkMDFiNTBlMGQxN2RjNzljOCI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiOTBmNzliZjZlYjJjNGY4NzAzNjVlNzg1OTgyZTFmMTAxZTkzYjkwNiI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiOTc2ZWE3NDAyNmU3MjY1NTRkYjY1N2ZhNTQ3NjNhYmQwYzNhMGFhOSI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiOTk2NTUwN2QxYTU1YmNjMjY5NWM1OGJhMTZmYjM3ZDgxOWIwYTRkYyI6IHsKICAgICAgImJhbGFuY2UiOiAiMHhkM2MyMWJjZWNjZWRhMTAwMDAwMCIKICAgIH0sCiAgICAiZjM5RmQ2ZTUxYWFkODhGNkY0Y2U2YUI4ODI3Mjc5Y2ZmRmI5MjI2NiI6IHsKICAgICAgImJhbGFuY2UiOiAiNjAwMDAwMDAwMDAwMDAwMDAwMDAwIgogICAgfSwKICAgICJmMzlmZDZlNTFhYWQ4OGY2ZjRjZTZhYjg4MjcyNzljZmZmYjkyMjY2IjogewogICAgICAiYmFsYW5jZSI6ICIweGQzYzIxYmNlY2NlZGExMDAwMDAwIgogICAgfQogIH0sCiAgImJhc2VGZWVQZXJHYXMiOiBudWxsLAogICJibG9iR2FzVXNlZCI6IG51bGwsCiAgImNvaW5iYXNlIjogIjB4MDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMCIsCiAgImNvbmZpZyI6IHsKICAgICJieXphbnRpdW1CbG9jayI6IDAsCiAgICAiY2hhaW5JZCI6IDQyLAogICAgImNvbnN0YW50aW5vcGxlQmxvY2siOiAwLAogICAgImR1cmFuZ29UaW1lc3RhbXAiOiAxNzE0NTI1MzI3LAogICAgImVpcDE1MEJsb2NrIjogMCwKICAgICJlaXAxNTVCbG9jayI6IDAsCiAgICAiZWlwMTU4QmxvY2siOiAwLAogICAgImZlZUNvbmZpZyI6IHsKICAgICAgImJhc2VGZWVDaGFuZ2VEZW5vbWluYXRvciI6IDM2LAogICAgICAiYmxvY2tHYXNDb3N0U3RlcCI6IDIwMDAwMCwKICAgICAgImdhc0xpbWl0IjogODAwMDAwMCwKICAgICAgIm1heEJsb2NrR2FzQ29zdCI6IDEwMDAwMDAsCiAgICAgICJtaW5CYXNlRmVlIjogMjUwMDAwMDAwMDAsCiAgICAgICJtaW5CbG9ja0dhc0Nvc3QiOiAwLAogICAgICAidGFyZ2V0QmxvY2tSYXRlIjogMiwKICAgICAgInRhcmdldEdhcyI6IDUwMDAwMDAwCiAgICB9LAogICAgImhvbWVzdGVhZEJsb2NrIjogMCwKICAgICJpc3RhbmJ1bEJsb2NrIjogMCwKICAgICJtdWlyR2xhY2llckJsb2NrIjogMCwKICAgICJwZXRlcnNidXJnQmxvY2siOiAwLAogICAgInN1Ym5ldEVWTVRpbWVzdGFtcCI6IDAsCiAgICAid2FycENvbmZpZyI6IHsKICAgICAgImJsb2NrVGltZXN0YW1wIjogMTcxNDUyNTQ5NiwKICAgICAgInF1b3J1bU51bWVyYXRvciI6IDY3CiAgICB9CiAgfSwKICAiZGlmZmljdWx0eSI6ICIweDAiLAogICJleGNlc3NCbG9iR2FzIjogbnVsbCwKICAiZXh0cmFEYXRhIjogIjB4IiwKICAiZ2FzTGltaXQiOiAiMHg3YTEyMDAiLAogICJnYXNVc2VkIjogIjB4MCIsCiAgIm1peEhhc2giOiAiMHgwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwIiwKICAibm9uY2UiOiAiMHgwIiwKICAibnVtYmVyIjogIjB4MCIsCiAgInBhcmVudEhhc2giOiAiMHgwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwMDAwIiwKICAidGltZXN0YW1wIjogIjB4MCIKfQ==",
```

Comment:
Can you give context to what this value is?

Reply:
this is the Genesis file for the avalanche lilypad testnet

Reply:
Ok, can you help me decipher the contents here? I do wish to trust but want to verify the contents of this genesis block that we will use in our testnet infra

Reply:
@Zorlin helped me realize this is `base64` and it actually is the genesis file ;)